subsys/power/device.c

```diff
@@ -60,26 +60,65 @@ static device_idx_t num_pm;
 /* Number of devices successfully suspended. */
 static device_idx_t num_susp;
 
+static bool should_suspend(const struct device *dev, uint32_t state)
+{
+	int rc;
+	uint32_t current_state;
+
+	rc = device_get_power_state(dev, &current_state);
+	if ((rc != -ENOTSUP) && (rc != 0)) {
+		LOG_DBG("Was not possible to get device %s state: %d",
```

Comment:
nit: `LOG_ERR`

Reply:
I can raise it to WRN but I don't think it should be ERR. ERR should be used for things that is not possible to recover